src/DrCabal/Profile/Format.hs

```diff
@@ -34,3 +37,6 @@ fmtNanos time
     emptyIfZero :: Word64 -> Text -> Text
     emptyIfZero 0 _    = ""
     emptyIfZero t unit = show t <> unit
+
+fmtDecimalPlaces :: Int -> Float -> Text
+fmtDecimalPlaces dp f = pack $ showFFloat (Just dp) f ""
```

Comment:
Cool usage of `shoFFloat`! Didn't know about this function, I used to use `printf` when I needed to format floating point numbers with fixed precision 🙂

Reply:
Yep, it’s very nice! I actually discovered it just today, when I wanted fixed precision but not any of the other features of `printf`. (In fact, it turns out that [`Numeric`](https://hackage.haskell.org/package/base-4.16.3.0/docs/Numeric.html) has quite a few useful little functions for doing these things.)